scripts/install-cni.sh

```diff
@@ -121,6 +121,20 @@ else
   cni_spec=${cni_spec//@cniCiliumPlugin/}
 fi
 
+## Look for optional env var to know if plug-in should be enabled
+if [[ -n "${ISTIO_CNI_CONFIG:-}" ]]; then
+  ENABLE_ISTIO_PLUGIN=true
```

Comment:
I'm not sure if we'd better have this, or just use `-n "${ISTIO_CNI_CONFIG:-}"` whenever we want to check `"${ENABLE_ISTIO_PLUGIN}" == "true"`.